### DISCUSSION TITLE
unmodifed Expedition 14 please?

### DISCUSSION THREAD

**Discussion by safiworks** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#9] [General]
duo to some issues i cannot download the expedition cache from the game directly can someone provide it for me? thanks

**Comment by cwmonkey** [comment]
Here it is as of the 5.01 patch:

[SEASON_DATA_CACHE.JSON](https://github.com/user-attachments/files/16391939/SEASON_DATA_CACHE.JSON)

Keep in mind this has community missions which cannot be completed offline. We will eventually make this available with those milestones replaced with ones which can be completed offline.

**Reply by safiworks** [reply]
thank you so much! you saved me 20 gigs of bandwidth (and my family from paying the internet services provider 5th time this month because we have a limit on bandwidth if exceeded you must pay)!

i will try to modify it myself
